docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -237,6 +236,7 @@ Do not use untagged or “latest”.
 		
 		- A full-sized sample is helpful for benchmarking your workflow and providing end-users with realistic compute and cost requirements. 
 
+- When writing your descriptor files, do not use remote imports (referencing software outside of the referenced container) or include scripts as input files. These practices decrease reusability and increases security risks. 
```

Comment:
Some researchers bring in, for example, R scripts as input files instead of writing out the R commands in a task. I think it's ok if the scripts are in the container though. Maybe:

"When writing your descriptor files, do not import remote descriptors using HTTP(s), nor use scripts outside of the container as input files."

Reply:
> "When writing your descriptor files, do not import remote descriptors using HTTP(s), nor use scripts outside of the container as input files."

That works for me.